packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -846,9 +846,19 @@ abstract class SliderComponentShape {
 ///  * [SliderTrackShape] Base component for creating other custom track
 ///    shapes.
 class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const RectangularSliderTrackShape();
+  /// Create a slider track that draws a circle 2 rectangles.
+  const RectangularSliderTrackShape({this.disabledThumbGap});
+
+  /// Spacing, or gap, between disabled thumb and the track.
+  ///
+  /// This is only used when the thumb is disabled. The material spec defaults
+  /// it to 2, which is half of its disabled thumb radius, which is 4.
```

Comment:
I think the `, which is 4.` is redundant. 

Consider:
```
/// This is only used when the thumb is disabled. The Material spec defaults
/// to 2, which is half of the disabled thumb radius.
```

Reply:
Done